apps/angular/injection-token/src/app/video.component.ts

```diff
@@ -1,9 +1,12 @@
 import { Component } from '@angular/core';
 import { TimerContainerComponent } from './timer-container.component';
+import { TIMER_TOKEN } from './data';
 
 @Component({
+  selector: 'app-video',
   standalone: true,
   imports: [TimerContainerComponent],
+  providers: [{ provide: TIMER_TOKEN, useValue: 1000 }],
```

Comment:
TIMER_TOKEN should have a default value to 1000, so this override should be avoided